crates/flox-rust-sdk/src/models/environment/managed_environment.rs

```diff
@@ -448,6 +473,55 @@ fn gcroots_dir(flox: &Flox, pointer: &ManagedPointer) -> PathBuf {
     flox.cache_dir.join("run").join(pointer.owner.to_string())
 }
 
+impl ManagedEnvironment {
+    #[allow(unused)]
+    pub fn pull(&mut self) -> Result<(), ManagedEnvironmentError> {
+        // Fetch the remote branch into FETCH_HEAD
+        self.floxmeta
```

Comment:
```suggestion
        // TODO just do a build in this method if open already did the pulling and locking
        self.floxmeta
```

Reply:
ill wait with building until we have building implemented for managed envs..